installer/scripts/omsadmin.sh

```diff
@@ -45,6 +45,8 @@ RESP_HEARTBEAT=$TMP_DIR/resp_heartbeat.xml
 BODY_RENEW_CERT=$TMP_DIR/body_renew_cert.xml
 RESP_RENEW_CERT=$TMP_DIR/resp_renew_cert.xml
 
+PROCESS_STATS=$TMP_DIR/process_stats
```

Comment:
I guess there's no way two instances can run at once? I'm wondering if you should make the filename $TMP_DIR/process_stats_$$ for uniqueness?